pom.xml

```diff
@@ -118,8 +120,8 @@
         <camel-community.version>4.8.0</camel-community.version>
         <camel-spring-boot-community.version>4.8.0</camel-spring-boot-community.version>
 
-        <camel-sap.version>4.0.0.redhat-00001</camel-sap.version>
-
+        <camel-sap.version>4.4.0.redhat-00016</camel-sap.version>
```

Comment:
is this version 4.4, correct?

Reply:
Yes, believe so.     We'll get a 4.8.0 productized version soon